app/src/sidecar.ts

```diff
@@ -1,12 +1,17 @@
-import axios, { AxiosInstance } from "axios";
+import axios, { AxiosInstance, AxiosStatic } from "axios";
 import { GetBlockResult } from "casper-js-sdk";
 
 import { SidecarDeploy } from "./types/api";
 
 export class Sidecar {
   private api: AxiosInstance;
 
-  constructor(public url: string) {
+  constructor(
+    public url: string,
+    private readonly tempDevNet: AxiosInstance = axios.create({
+      baseURL: "http://jakub.devnet.casperlabs.io:8888",
```

Comment:
My thought reading this:

Greg: Can that be put in app-config?

Reply:
My bad, I should have been more explicit with my intention for this epic (or at least left a comment). This is a devnet test endpoint developed by Jakub for the time being. He will integrate this method into the current sidecar protocol when it's ready so we'll just be using that long term. For now I've just left a comment.

Reply:
All good my man!  Thanks for the context